core/src/main/resources/reference.conf

```diff
@@ -351,6 +351,13 @@ cassandra-journal {
   # Enable JMX reporter for Dropwizard Metrics.
   # See: https://docs.datastax.com/en/developer/java-driver/3.5/manual/metrics/
   jmx-reporting-enabled = on
+
+  # If there is an unexpected error in the Cassandra Journal the journal shutsdown
+  # to prevent any data corruption. Set to true to also run coordinated shutdown so
+  # that the application shuts down. Useful in environemtns where applicatons are
+  # automatically shutdown or if the Cassandra Journal not working means the application
+  # won't function
```

Comment:
shutsdown => shuts down
environemtns => environments
period in the end of the last sentence

Reply:
baby brain